airbyte-integrations/connectors/source-braintree-singer/source_braintree_singer/spec.json

```diff
@@ -0,0 +1,42 @@
+{
+  "documentationUrl": "https://docs.airbyte.io/integrations/sources/braintree",
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Braintree Source Spec",
+    "type": "object",
+    "required": [
+      "merchant_id",
+      "public_key",
+      "start_date",
+      "private_key",
+      "environment"
+    ],
+    "additionalProperties": false,
+    "properties": {
+      "merchant_id": {
+        "type": "string",
+        "description": "<a href=\"https://articles.braintreepayments.com/control-panel/important-gateway-credentials#merchant-id\">Merchant ID</a> is the unique identifier for entire gateway account."
```

Comment:
we don't need `<a>` tags on any of these fields since they're described in the docs. Can you include a link to the airbyte Braintree doc on one of these but not the rest of the descriptions? In general it's preferable to link to the Airbyte docs in the spec and not the braintree docs because we can control those URLs.